includes/ApplHandlingByStr.php

```diff
@@ -69,6 +70,7 @@ function showFullStatus( $stat_str )
 		else if( $str[1] == 'R' )
 		{
 			echo "<td><b>REJECTED</b> by ".$str[0] ."</td>";
+			break;
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->